src/pages/logbook/[id].tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+
+import LogbookDetail from "~/views/Logbook/LogbookDetail"
+
+type PageProps = {
+  params: Record<string, any>
+}
+
+const LogbookDetailPage: React.FC<PageProps> = ({ params }) => (
+  <LogbookDetail id={params.id} originalPath={`/logbook/${params.id}`} />
```

Comment:
pageContext has originalPath but its value is `/logbook/[id]` which we can't use